themes/frontierline/sass/components/page-sections/_sidebar-custom.scss

```diff
@@ -0,0 +1,35 @@
+.custom-sidebar {
+  border: 5px solid #000;
+  box-sizing: border-box;
+  padding: 20px;
+
+  @media #{$mq-tablet} {
+    float: left;
+    margin-left: 5%;
+    width: 35%;
```

Comment:
This is the only requested change that didn't quite work. Gutters were too narrow because of my use of a border. I can add a nested div, or can leave as is. Let me know if you'd prefer that and I'll update.